Sources/TextureView/TextureView.swift

```diff
@@ -107,19 +106,18 @@ public class TextureView: UIView {
     }
 
     // MARK: - Setup
-    
+
     private func commonInit() {
         self.layer.device = self.device
         self.layer.framebufferOnly = true
         self.layer.isOpaque = false
-        self.layer.maximumDrawableCount = 3
```

Comment:
3 is the default value - I checked. 
Since in the first version you used value 2, maybe it even makes sense to pass `maximumDrawableCount` as a parameter in `init` with the default value of 3?

Reply:
I think we should make a public property and remove hidden customisation from init

Reply:
Another thought: `self.layer` itself is a public property. It means that all its properties are already available to users. So I think it would be confusing to introduce a separate property `TextureView.maximumDrawableCount` if users can change it it via `textureView.layer.maximumDrawableCount` anyway.

Reply:
agree